common/utils/src/main/resources/error/error-conditions.json

```
@@ -4634,6 +4634,18 @@
     ],
     "sqlState" : "42K09"
   },
+  "TUPLE_IS_EMPTY" : {
+    "message" : [
+      "Due to Scala's limited support of tuple, empty tuple is not supported."
+    ],
+    "sqlState" : "22004"
+  },
+  "TUPLE_SIZE_EXCEEDS_LIMIT" : {
+    "message" : [
+      "Due to Scala's limited support of tuple, tuple with more than 22 elements are not supported."
```

either tuple -> tuples or are -> is
```suggestion
      "Due to Scala's limited support of tuple, tuples with more than 22 elements are not supported."
```

---

Applied the suggestion. Thanks!